workflows/svclustering.nf

```diff
@@ -9,13 +9,34 @@ include { softwareVersionsToYAML      } from '../subworkflows/nf-core/utils_nfco
 include { PREPROCESSING               } from '../modules/local/preprocessing/preprocessing.nf'
 include { SVCLUSTERINGDUP             } from '../modules/local/svclustering/svclusteringdup.nf'
 include { SVCLUSTERINGDEL             } from '../modules/local/svclustering/svclusteringdel.nf'
-
+include { BCFTOOLS_VIEW  as FILTER    } from '../modules/nf-core/bcftools/view/main.nf'
 /*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     RUN MAIN WORKFLOW
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
+def groupSamples(input_ch){
+    return input_ch.map { meta, vcf ->
+        def fnum = 1
+        [fnum, [meta.familyId] + meta.sample, meta.sample, vcf]
+    }
+    .groupTuple()  // Group by familyId
+}
+
+def filterVariants(input_channel, include_only_pass_variants) {
+    if (!include_only_pass_variants) {
+        return input_channel
+    }
+    ch_filter_input =  input_channel.map{ meta, vcf -> 
+        def tbi = file(vcf + ".tbi")
+        tbi.exists() ?
+        [meta, vcf, tbi] :
+        [meta, vcf, []]
```

Comment:
Is the index is optional for this step? If so is it also optional for the next ones? If not, it might be good to produce an error if the index is missing here (or earlier)